.github/ISSUE_TEMPLATE/ad-hoc-meeting.yaml

```
@@ -0,0 +1,51 @@
+name: 'Create an ad-hoc meeting'
+description: 'This issue template is used for creating a new ad-hoc meeting for the Cross Project Council'
+title: "[Ad hoc meeting] "
+assignees:
+  - bensternthal
+labels:
+  - cross-project-council-agenda
+body:
+  - type: markdown
+    attributes:
+      value: |
+        ### Agenda for the Meeting
+  - type: input
+    id: facilitator
+    attributes:
+      label: Facilitator
+      description: Who is the facilitator of this meeting?
+    validations:
+      required: true
+  - type: input
+    id: related-issues
+    attributes:
+      label: Related issue(s)
+      description: What are the related issues to this meeting?
+    validations:
+      required: false
+  - type: input
+    id: meeting-date
+    attributes:
+      label: Date and time
+      description: 'When should the meeting happen? Use a "YYYY-MM-DD HH:MM UTC" format (e.g.: 2009-05-27 13:00 UTC)'
+    validations:
+      required: true
+  - type: input
+    id: zoom-link
+    attributes:
+      label: Zoom link
+      description: Where should the meeting happen?
+      value: 'https://zoom.us/j/91398697172?pwd=QVgxRUFXdmtHVUNiODhoSU11ejF0QT09'
+    validations:
+      required: true
+  - type: input
+    id: meeting-notes
+    attributes:
+      label: URL for meeting notes
+      description: Provide link to meeting notes
```

Suggest: "Provide link to Agenda"

---

Hi, @bensternthal. Purely FYI, and apologies if you already know this and are intentionally avoiding using it for a particular reason, but if you mark up your suggestion like this:

````markdown
```suggestion
      description: Provide link to Agenda
```
````

...it will render like this with a convenient button for the PR author and everything:

```suggestion
      description: Provide link to Agenda
```

---

I did not know that I could do this!! Thanks for the tip!